### ISSUE TITLE
(Android) ipega bluetooth controller issues

### ISSUE THREAD

**Issue by RogueMoon** [libretro/RetroArch#184]
On GB, using bluezIME to connect.
iCade mode is a no-go, with or without configuration autodetect.
Keyboard and joystick mode - buttons are detected in custom binds, and
I can even navigate the menus with the dpad, however no button
seems to respond within the games themselves (and nothing appears on screen
when choosing debug input reporting).

**Comment by Davidcalmops** [comment]
Get APK GBA emulator on playstore. Go to gamepad in menu then virtual then set buttons. It will remap buttons for you. It worked for me.

**Comment by mikoge** [comment]
In my case, im running android :) , the 1st time i run retroarch  , it detected my gamepad as an amazon gamepad and every buttons seem to work fine while i went downloading the cores and etc...but after closing it and the second time I opened retroarch it actually recognized the real model of my ipega-9021 but the weird and most annoying thing is, no buttons are working anymore and retroarch looks frozen or hangs. Its giving me a headache . I havent been able to play any game. 
I like to play Dreamcast, I tried installing flycast as stand alone , almost everything works except it doesn't save my games , might be bugs from saving to vmu. Have to play mvsc2 over and over again with less characters
